WoWsShipBuilder.Web/Dialogs/HideComparisonColumnsDialog.razor

```diff
@@ -0,0 +1,79 @@
+﻿@using WoWsShipBuilder.ViewModels.Other
+@inject ILocalizer Localizer
+
+<MudDialog Style="max-height: 95vh" DefaultFocus="DefaultFocus.Element">
+    <DialogContent>
+        <div style="max-height: 85vh; overflow: scroll;">
+            <MudText Typo="Typo.body2" Align="Align.Center" Style="font-weight: bold" Class="mb-1">@Localizer.SimpleAppLocalization(nameof(Translation.ShipComparison_ActiveColumns)).ToUpper()</MudText>
+            <MudStack Row>
+                @foreach (var (section, columns) in structure)
+                {
+                    <MudPaper Outlined Elevation="0" Square="true" @key="@(section.ShipComparisonDataSectionToString())">
+                        <MudStack Justify="Justify.Center" AlignItems="AlignItems.Center">
+                            <MudText Typo="Typo.body2" Align="Align.Center" Style="font-weight: bold">
+                                @(section.LocalizeShipComparisonDataSection(Localizer).ToUpper())
+                            </MudText>
+                            @foreach (var column in columns)
+                            {
+                                <MudSwitch @bind-Checked="@(structure[section][column.Key])" Label="@column.Key" LabelPosition="LabelPosition.End" Size="Size.Medium" Color="Color.Warning" @key="@($"{section}_{column}")" Style="width: 100%;" Class="ml-1"/>
+                            }
+                        </MudStack>
+                    </MudPaper>
+                }
+            </MudStack>
+        </div>
+    </DialogContent>
+    <DialogActions>
+        <MudButton OnClick="Cancel" Variant="Variant.Text">@Localizer.GetAppLocalization(nameof(Translation.Cancel)).Localization</MudButton>
+        <MudButton OnClick="Confirm" Variant="Variant.Text">@Localizer.GetAppLocalization(nameof(Translation.Dialog_Save)).Localization</MudButton>
+    </DialogActions>
+</MudDialog>
+
+@code
+{
+    [CascadingParameter]
+    MudDialogInstance MudDialog { get; set; } = new();
+
+    [Parameter, EditorRequired]
+    public Dictionary<string, bool> HiddenColumns { get; set; } = new();
+
+    private readonly Dictionary<ShipComparisonDataSections, Dictionary<string, bool>> structure = new();
```

Comment:
Change the name to something that is more clear about what it does. "structure" does not say anything about its purpose